task/sql-tasks.js

```diff
@@ -223,7 +360,16 @@ async function task_1_16(db) {
  *
  */
 async function task_1_17(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT 
+            c.CategoryName, avg(p.UnitPrice) as AvgPrice
+        FROM northwind.Categories as c
+        right JOIN northwind.Products as p

```

Comment:
do not use right join if not necessary. Prefer left join if possible

Reply:
fixed bugs